hack/e2e-common.sh

```diff
@@ -220,6 +221,18 @@ case "${CLOUD}" in
 	CREDS_FILE_ARG="--creds-file=${CLUSTER_PROFILE_DIR}/gce.json"
 	BASE_DOMAIN="${BASE_DOMAIN:-origin-ci-int-gce.dev.openshift.com}"
 	;;
+"vsphere")
+  BASE_DOMAIN="${BASE_DOMAIN:-vmc.devcluster.openshift.com}"
+  API_VIP=$(get_vips 3) # Get 3rd vip from file
+  INGRESS_VIP=$(get_vips 4) # Get 4th vip from file
+  EXTRA_CREATE_CLUSTER_ARGS="--vsphere-datacenter=${GOVC_DATACENTER:-DEVQEdatacenter} \
+      --vsphere-default-datastore=${GOVC_DATASTORE:-vsanDatastore}\
+      --vsphere-cluster=${VSPHERE_CLUSTER:-DEVQEcluster}
+      --vsphere-api-vip=$API_VIP \
+      --vsphere-ingress-vip=$INGRESS_VIP \
+      --vsphere-network=$NETWORK_NAME \
+      --vsphere-vcenter=$VCENTER"
```

Comment:
Relies on two variables that we're not defaulting or validating. I don't love `set -o nounset`, but maybe add some explicit validation for them? Just to make debugging easier.